source/Concepts/Basic/About-Launch.rst

```diff
@@ -7,30 +7,48 @@ Launch
 Overview
 --------
 
-A ROS 2 system typically consists of multiple nodes running across different processes (and even different machines).
-While it is possible to run each of these nodes separately, it can get cumbersome rather quickly.
+A ROS 2 system typically consists of multiple :doc:`nodes <About-Nodes>` running across different processes (and even different machines).
+While it is possible to start each of these nodes manually, it can get cumbersome rather quickly.
+Automation is also non-trivial, even if existing tools for generic process control and configuration management are leveraged, let alone crafting it from first principles (e.g. proper OS process management using shell scripts or general purpose system programming languages can be tricky on its own).
 
-The purpose of the ``launch`` system in ROS 2 is to provide the means to orchestrate the execution of such a system. This tool uses _launch files_ to describe the system and its configuration, but not limited to which programs to run, how to run them and which arguments to use for them. These descriptions are then executed, starting processes, monitoring them, reporting on their state, and even reacting to their behavior.
+``launch`` provides the means to orchestrate the execution of ROS 2 systems, in ROS 2 terms.
+_Launch files_ describe the orchestration procedure, including but not limited to which nodes to run, how to run them, and which arguments to use for them.
+These descriptions are then executed, starting processes, monitoring them, reporting on their state, and even reacting to their behavior.
 
-System descriptions
--------------------
+Moving parts
```

Comment:
Would it be possible to link each of the defined words to the existing documentation / tutorials for them?

Reply:
I don't think this is documented with this granularity outside launch API documentation. I can try though.

Reply:
Well, I ended up pointing to `launch` API docs in https://github.com/mhidalgo-bdai/ros2_documentation/pull/1/commits/1ecbbac0e4a1339cc85886fb9a60f50c9ede4214. It's closest there is out there.